core/app/models/spree/google_export_option.rb

```diff
@@ -0,0 +1,19 @@
+module Spree
+  class GoogleExportOption < Base
```

Comment:
I saw in `migration` file field related to `Spree::Store` should we add here relationship with this model ?

Reply:
yeah, that would make sense, to make sure it refers to a store that actually exists